src/main/java/com/google/cloud/genomics/dataflow/pipelines/LoadReadsToBigQuery.java

```diff
@@ -0,0 +1,372 @@
+          .set(FRAGMENT_LENGTH, c.element().getFragmentLength())
+          .set(READ_NUMBER, c.element().getReadNumber())
+          .set(NUMBER_READS, c.element().getNumberReads())
+          .set(FAILED_VENDOR_QUALITY_CHECKS, c.element().getFailedVendorQualityChecks())
+          .set(SECONDARY_ALIGNMENT, c.element().getSecondaryAlignment())
+          .set(SUPPLEMENTARY_ALIGNMENT, c.element().getSupplementaryAlignment())
+          .set(ALIGNED_SEQUENCE, c.element().getAlignedSequence())
+          .set(ALIGNED_QUALITY, c.element().getAlignedQualityList())
+          .set(NEXT_MATE_POSITION,
+              new TableRow().set(REFERENCE_NAME,
+                                 c.element().getNextMatePosition().getReferenceName())
+                            .set(POSITION, c.element().getNextMatePosition().getPosition())
+                            .set(REVERSE_STRAND,
+                                 c.element().getNextMatePosition().getReverseStrand())
+               );
+      // Alignment
+      TableRow alignmentPosition =
+          new TableRow().set(REFERENCE_NAME,
+                             c.element().getAlignment().getPosition().getReferenceName())
+                        .set(POSITION, c.element().getAlignment().getPosition().getPosition())
+                        .set(REVERSE_STRAND,
+                             c.element().getAlignment().getPosition().getReverseStrand());
+      List<TableRow> alignmentCigars = new ArrayList<>();
+      for (CigarUnit cigar : c.element().getAlignment().getCigarList()) {
+        TableRow alignmentCigar =
+            new TableRow().set(OPERATION, cigar.getOperation().getNumber())
+                          .set(OPERATION_LENGTH, cigar.getOperationLength())
+                          .set(REFERENCE_SEQUENCE, cigar.getReferenceSequence());
+        alignmentCigars.add(alignmentCigar);
+      }
+      TableRow alignmentRecord = new TableRow().set(POSITION, alignmentPosition)
+          .set(MAPPING_QUALITY, c.element().getAlignment().getMappingQuality())
+          .set(CIGAR, alignmentCigars);
+      row.set(ALIGNMENT, alignmentRecord);
+
+      c.output(row);
+    }
+  }
+
+  public static void main(String[] args) throws GeneralSecurityException, IOException,
+         URISyntaxException {
+    // Register the options so that they show up via --help.
+    PipelineOptionsFactory.register(Options.class);
+    pipelineOptions = PipelineOptionsFactory.fromArgs(args)
+        .withValidation().as(Options.class);
+    // Option validation is not yet automatic, we make an explicit call here.
+    Options.Methods.validateOptions(pipelineOptions);
+    // Needed for BigQuery.
+    pipelineOptions.setTempLocation(pipelineOptions.getTempPath());
+
+    auth = GenomicsOptions.Methods.getGenomicsAuth(pipelineOptions);
+    p = Pipeline.create(pipelineOptions);
+
+    // Ensure data is accessible.
+    String BAMFilePath = pipelineOptions.getBAMFilePath();
```

Comment:
Java style is that initialisms are spelled Bam not BAM (so BamFilePath) and GCS -> Gcs